dlrover/python/master/monitor/speed_monitor.py

```diff
@@ -142,6 +142,9 @@ def reset_running_speed_monitor(self):
         """Reest the speed monitor"""
         self._global_step_records = []
 
+    def achieve_worker_target(self):
```

Comment:
`achieve_worker_target` is used to check that all worker are running and starting to get shard from master?
Can it be renamed as `workers_ready`, `all_workers_joined`   or something else so that it is more literal?

Reply:
`all_workers_joined` is good!